.github/workflows/pr-test.yml

```diff
@@ -41,6 +41,7 @@ jobs:
 
       # Options
       standard: true
+      minimum-base-package: true
```

Comment:
Let's revert this here and verify in a different PR with the approach that we discussed in internal slack